### PR TITLE
Add notes for System.CommandLine review

### PR DESCRIPTION
**Pull request by terrajobst**
Notes for today

### CODE REVIEW

File: 2020/01-21-command-line/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - Still iterating
+  - Refactoring to smaller pieces for BCL/Not BCL not helpful
+* Today: 
+  - User study
+
+## Notes
+
+* Handler should probably not use delegates and instead take a raw method info
+  if the implementation relies on the parameter names coming from the user
+  defined method. The compiler might create a wrapper delagate which would
```
Comment (jkotas) [inline 2020/01-21-command-line/README.md]:
```suggestion
  defined method. The compiler might create a wrapper delegate which would
```

File: 2020/01-21-command-line/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# System.CommandLine
+
+## Pieces: 
+
+* Parser and invocation
+* Console
+
+## Console:
+
+* Ansi Terminal/VT 100 - System.Console has limited and inconsistent support
```
Comment (stephentoub) [inline 2020/01-21-command-line/README.md]: Is there a more detailed list of what's missing and what's inconsistent?

Comment (terrajobst) [reply 2020/01-21-command-line/README.md]: @jonsequitur, do you have a list?

Comment (terrajobst) [reply 2020/01-21-command-line/README.md]: @jonsequitur, do you have a list?

**Pull request merged**